zhazam/Modules/StartGameTimer/CountdownViewController.swift

```diff
@@ -49,16 +47,16 @@ final class StartGameTimerViewController: UIViewController {
 //                                  animations: {},
 //                                  completion: nil)
         
-        //make transition to Game page
+//        make transition to Game page
     }
     
-    private func makeTimer(completion: @escaping() -> Void) {
+    private func configureCountdownTimer(completion: @escaping() -> Void) {
         var rowIndex = 0
         Timer.scheduledTimer(withTimeInterval: Constants.timerTimeInterval, repeats: true) { timer in
-            self.timerPickerView.selectRow(rowIndex, inComponent: 0, animated: true)
+            self.pickerView.selectRow(rowIndex, inComponent: 0, animated: true)
             rowIndex += 1
             
-            if Constants.counterArray.count <= rowIndex - Constants.lastRowDisplayIndex {
+            if Constants.time < rowIndex {
```

Comment:
still not clear, I didn't mean to shorten this condition, you just had to move to a variable (Write me if you did not get the idea)

Reply:
remove "time" constant,  bring your initial condition back